src/components/Navigation/MobileNavigation/index.js

```diff
@@ -0,0 +1,234 @@
+    borderRadius: typography.pxToRem(50),
+    margin: typography.pxToRem(8),
+    "&:hover": {
+      background: "#F0F0F0",
+      borderRadius: typography.pxToRem(50),
+      margin: typography.pxToRem(8),
+    },
+  },
+  closeButton: {
+    color: palette.background.main,
+    width: 48,
+    paddingLeft: typography.pxToRem(12),
+    "&:hover": {
+      background: "none",
+    },
+  },
+  menuItems: {
+    padding: `${typography.pxToRem(20)} 0 ${typography.pxToRem(71)}`,
+  },
+  button: {
+    color: palette.background.dark,
+    padding: typography.pxToRem(16),
+  },
+  open: {
+    fontSize: typography.pxToRem(32),
+  },
+  close: {
+    fontSize: typography.pxToRem(32),
+    color: palette.background.default,
+  },
+}));
+
+const Transition = React.forwardRef(function Transition(props, ref) {
+  return <Slide direction="left" timeout={1000} ref={ref} {...props} />;
+});
+
+function MobileNavigation({
+  logoProps,
+  menuProps,
+  selectProps,
+  socialLinks,
+  ...props
+}) {
+  const classes = useStyles(props);
+  const [open, setOpen] = useState(false);
+
+  const handleClickOpen = (e) => {
+    e?.preventDefault();
+    setOpen(true);
+  };
+  const handleClose = (e) => {
+    e?.preventDefault();
+    setOpen(false);
+  };
+  return (
+    <Section classes={{ root: classes.section }}>
+      <div className={classes.root}>
+        <Grid item>
+          <Logo {...logoProps} />
+        </Grid>
+        <Grid item>
+          <IconButton
+            aria-label="Open drawer"
+            edge="start"
+            size="medium"
+            onClick={handleClickOpen}
+            className={classes.menuButton}
+          >
+            <Image
+              src={MenuOpenIcon}
+              width={48}
+              height={48}
+              className={classes.open}
+            />
+          </IconButton>
+        </Grid>
+        <Dialog
+          fullScreen
+          open={open}
+          onClose={handleClose}
+          BackdropProps={{
+            classes: {
+              root: classes.backdrop,
+            },
+          }}
+          TransitionComponent={Transition}
+          classes={{ root: classes.dialog, paper: classes.dialogPaper }}
+        >
+          <DialogActions className={classes.dialogActions}>
+            <Grid container direction="row" className={classes.logoSection}>
+              <Grid
+                item
+                xs={8}
+                container
+                justifyContent="flex-start"
+                alignItems="center"
+                className={classes.dialogMenu}
+              >
+                <Logo
+                  {...logoProps}
+                  classes={{
+                    firstTitle: classes.firstTitle,
+                    secondTitle: classes.secondTitle,
+                    subtitle: classes.subtitle,
+                  }}
+                />
+              </Grid>
+
+              <Grid
+                item
+                xs={4}
+                container
```

Comment:
Same here, a single item can reside inside a `Grid item`... Only when you need multiple items that's when you make the `Grid item` a `container` as well.

I take it you wanted the logo and close buttons to be as far apart as possible? Use justify="space-between" on their parent `Grid container`.